### PR TITLE
MSI - Add cluster install options

### PR DESCRIPTION
**Pull request by rnetser**
Requires https://github.com/openshift/release/pull/41105

### CODE REVIEW

**Comment by rnetser** [comment]
/pj-rehearse  pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse  pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-aws

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-aws

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-aws

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-rosa

**Comment by rnetser** [comment]
pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-rosa-smoke

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-smoke

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Review by myakove — APPROVED** [review]

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-smoke

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-smoke

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-production-cluster-install-uninstall-rosa

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-uninstall-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-aws-ipi

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-multi-cluster-install-aws

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-production-cluster-install-uninstall-rosa

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-cluster-install-uninstall-hypershift

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-rosa

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-v4.13-stage-cluster-install-uninstall-rosa

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-cluster-install-uninstall-rosa

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-smoke

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-cluster-install-uninstall-cluster-install-uninstall-rosa

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-multi-cluster-smoke-v4.13-multi-cluster-aws-hypershift-smoke

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-cluster-install-uninstall.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      PULL_SECRET_NAME: pull-secret
+      S3_BUCKET_NAME: openshift-cli-installer
+      S3_BUCKET_PATH: openshift-ci
+    test:
+    - ref: cluster-install
+    - ref: cluster-uninstall
+zz_generated_metadata:
+  branch: cluster-install-uninstall
+  org: CSPI-QE
+  repo: MSI
```
Comment (myakove) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-cluster-install-uninstall.yaml:72]: Do we want to add `variant: v4.13`?

Comment (rnetser) [reply ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-cluster-install-uninstall.yaml:72]: The test includes 4.13 and 4.14

**Pull request merged**